app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/LibreUtils.java

```diff
@@ -1,10 +1,14 @@
 package com.eveningoutpost.dexdrip.UtilityModels;
 
+import com.eveningoutpost.dexdrip.Home;
+
 import android.util.Log;
 
-public class LibreCrc {
+public class LibreUtils {
+
+    private static final String TAG = LibreUtils.class.getSimpleName();
 
-    private static final String TAG = LibreCrc.class.getSimpleName();
+    private static final boolean testWithDeadSensor = false; // never in production
```

Comment:
Can we add `@Getter` annotation to `testWithDeadSensor` so that it can be referenced elsewhere? For example to bypass the clock advancement check and so that something can be displayed in the UI to warn that dead sensors are being allowed. Then I could make this a run time configuration option for engineering mode.